### DISCUSSION TITLE
Can't access GitPod

### DISCUSSION THREAD

**Discussion by OfarLouis** [DevSkillsHQ/discusssions#45] [Help]
Hello, I am using the repository ffc1e12/lookup-service-level-1_7ab7a9d-f2gd49 and trying to use it with GitPod, however it says 404 not found. Any advice would be greatly appreciated!

**Comment by fenske** [comment]
Hi @OfarLouis. I'm Anton from Alva. That's odd. 🤔  Sounds like a permission issue.

Could you please attach a screenshot (or screencast ideally) of what happens when you navigate to [http://gitpod.io/#https://github.com/ffc1e12/lookup-service-level-1_7ab7a9d-f2gd49](http://gitpod.io/#https://github.com/ffc1e12/lookup-service-level-1_7ab7a9d-f2gd49)

**Reply by fenske** [reply]
Or, if I may suggest an alternative, that'd be Codespaces that we're actually moving to as Gitpod sometimes causes issues like this.

Here's how you spin it up: https://www.loom.com/share/6f83c8e1208745e395bea3b08594cf9d

Please let me know how it went!
